analyzer/quality-analyzer/src/main/java/eu/fasten/analyzer/qualityanalyzer/QualityAnalyzerPlugin.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package eu.fasten.analyzer.qualityanalyzer;
+
+import eu.fasten.analyzer.qualityanalyzer.data.QAConstants;
+
+import eu.fasten.core.plugins.KafkaPlugin;
+import eu.fasten.core.plugins.DBConnector;
+
+import org.json.JSONObject;
+import org.pf4j.Extension;
+import org.pf4j.Plugin;
+import org.pf4j.PluginWrapper;
+
+import org.slf4j.LoggerFactory;
+import org.slf4j.Logger;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.List;
+import java.util.Collections;
+
+import org.jooq.DSLContext;
+
+
+public class QualityAnalyzerPlugin extends Plugin {
+
+    public QualityAnalyzerPlugin(PluginWrapper wrapper) {
+        super(wrapper);
+    }
+
+    @Extension
+    public static class QualityAnalyzer implements KafkaPlugin, DBConnector {
+
+        private final Logger logger = LoggerFactory.getLogger(QualityAnalyzer.class.getName());
+        private String consumerTopic = "fasten.RapidPlugin.out";
+        private MetadataUtils utils = null;
+
+        @Override
+        public void setDBConnection(Map<String, DSLContext> dslContexts) {
+            this.utils = new MetadataUtils(dslContexts);
```

Comment:
Please make `utils` static and here use `QualityAnalyzer.utils = new MetadataUtils(dslContexts)`. Otherwise you will get a `NullPointerException` when running this plugin with FASTEN Server

Reply:
OK, done.

Reply:
OK, done